recipes/hepmc2/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hepmc2" %}
+{% set version = "2.06.07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lcgapp.cern.ch/project/simu/HepMC/download/HepMC-{{ version }}.tar.gz
+  sha256: a0bdd6f36a3cc4cb59d6eb15cef9d46ce9b3739cae3324e81ebb2df6943e4594
+  folder: source
+  patches:
+    - patches/hepmc-2.03.06-reflex.patch
+    - patches/hepmc-2.06.07-WeightContainer-fix-size_type.patch
+    - patches/HepMC-2.06.07-nodoc.patch
+    - patches/HepMC-2.06.07-broken-tests.patch
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+
+  run:
+
+test:
+  source_files:
+    - source/
+  requires:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
It's nice to have some kind of test to make sure the package actually works after being relocated. How about using this for testing?
```suggestion
  commands:
    # Workaround for lack of CONDA_BUILD_SYSROOT variable in staged recipes
    - export CONDA_BUILD_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk  # [osx]
    - cd source/examples
    # example_UsingIterators
    - ${CXX} example_UsingIterators.cc -o example_UsingIterators -I${CONDA_PREFIX}/include -lhepmc
    # Fix up the runpath on macOS
    - ${INSTALL_NAME_TOOL} -change @rpath/libHepMC.4.dylib $CONDA_PREFIX/lib/libHepMC.4.dylib example_UsingIterators # [osx]
    - ./example_UsingIterators
    # example_EventSelection
    - ${CXX} example_EventSelection.cc -o example_EventSelection -I${CONDA_PREFIX}/include -lhepmc
    # Fix up the runpath on macOS
    - ${INSTALL_NAME_TOOL} -change @rpath/libHepMC.4.dylib $CONDA_PREFIX/lib/libHepMC.4.dylib example_EventSelection # [osx]
    - ./example_EventSelection
    # example_BuildEventFromScratch
    - ${CXX} example_BuildEventFromScratch.cc -o example_BuildEventFromScratch -I${CONDA_PREFIX}/include -lhepmc
    # Fix up the runpath on macOS
    - ${INSTALL_NAME_TOOL} -change @rpath/libHepMC.4.dylib $CONDA_PREFIX/lib/libHepMC.4.dylib example_BuildEventFromScratch # [osx]
    - ./example_BuildEventFromScratch
```